code/modules/client/preferences_savefile.dm

```diff
@@ -127,12 +127,8 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 		if(S["facial_style_name"])
 			S["facial_style_name"]	>> facial_hairstyle
 			
-	if(current_version < 26)
-		key_bindings = sanitize_islist(key_bindings, deepCopyList(GLOB.keybinding_list_by_key))
-		key_bindings["T"] = list(1 = "say")
-		key_bindings["M"] = list(1 = "me")
-		key_bindings["O"] = list(1 = "ooc")
-		key_bindings["L"] = list(1 = "looc")
+	if(current_version < 27)
+		key_bindings = deepCopyList(GLOB.keybinding_list_by_key)
```

Comment:
this should reset everyone who is below version 27's keybindings to default, should clean up the weird ones added in version 26

Reply:
Best to nuke it from orbit, only way to be sure.